### ISSUE TITLE
Cannot load local image files

### ISSUE THREAD

**Issue by lightwithoutheat** [Telerik-Verified-Plugins/WKWebView#183]
I have an app that displays images stored on the local device file system, e.g. 
<img ng-src="{{loadLocalFile(name)}}">
function loadLocalFile(name) {  return cordova.file.dataDirectory + name; }

This has all been working perfectly, until I installed the WkWebView plugin and now my images are no longer loading.  Is there anything obvious that I've missed or that needs to be added after installing this plugin or is this a bug?

**Comment by mrsubtle** [comment]
Possibly related, but my project with a LaunchScreen.xib that includes an Image control pointing to a local image file is now broken as well.

**Comment by leobetosouza** [comment]
Same here. `cordova.file.applicationStorageDirectory` fails since installation of WKWebView

**Comment by dylanspurgin** [comment]
Just to hopefully allow people to work around this issue, we were able to use cordova-file-plugin to read files once we started using WKWebView. We had to use applicationStorageDirectory for iOS and externalApplicationStorageDirectory for Android (we are reading images taken from the camera). We read the file as base64 encoded data and were able to pass that string back to ng-src (AngularJs image src attribute) to display the image.

**Comment by mrsubtle** [comment]
Thanks for the workaround suggestion. Much appreciated. 

Richard Bennett
rich@n2d.co   |   http://n2d.co/   |   @mr_subtle   |   Behance   |   Dunked   |   Scoutzie

**Comment by castrojr913** [comment]
@dylanspurgin Yeah! the trick is using the image with Base 64 format. I had a similar issue with cordova camera plugin which my iOS application crashed trying to read an local image and this approach fixed the trouble.
